UI/hotkey-edit.cpp

```diff
@@ -293,11 +293,13 @@ void OBSHotkeyWidget::AddEdit(obs_key_combination combo, int idx)
 
 	auto add = new QPushButton;
 	add->setProperty("themeID", "addIconSmall");
```

Comment:
Docs claim this only sets the fallback AccessibleDescription, should the AccessibleName also be set?

Reply:
At least with NVDA running with defaults, it would always read both the name and the description if they're available, and if you set the name, the tooltip still gets read for the description part. In this instance, setting the accessibleName to "Add" would result in NVDA reading out "Add Add"

Reply:
Should name be removed from the elements where both are specified then? Or otherwise unified into description/name whichever you feel more appropriate.

Reply:
Name and description are said one after the other if there isn't some state to be reported. The full order appears to be "Name, UI Type, Value, Focused, Description". UI Type is not always used (for example, not used for buttons but for checkboxes will say "CheckBox"), and focused is said only used on some cases, and does not mention when something is unfocused.

Reply:
In this specific instance, the button doesn't get it's type mentioned, doesn't have a value to report, and will only report focus if you explicitly give it to it. As such, Name and Description end up said one after the other. In other cases, the name and description would be separated.

Reply:
Sorry maybe i wasnt clear, I was referring to Visibility/Lock for sources. They have a verbose description and name so it seemed like this section should too. Otherwise if only a terse name is fine should visibility/lock also be as terse? 

Also If description is read every time is a terse name more preferable to sources where the elements are repeated once per source?

Thanks for taking the time to describe how this ends up working in a real screen reader.

Reply:
The hotkeys section in general has a bunch of other fixes that need doing. The labels and edit boxes have tooltips if they're paired with another hotkey to explain how it can become a toggle, and that ends up getting read out everytime you select the label or edit box with the screen reader. I also didn't want to add too many new locales into this PR given v24 is really close. The change I made works using just existing locales.

The visibility/lock stuff required a new locale regardless, so I made it as descriptive as I felt it should be.

Reply:
I see thanks.